packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -51,25 +57,57 @@ export default function BlockToolbar( { hideDragHandle } ) {
 		};
 	}, [] );
 
-	if ( blockClientIds.length === 0 ) {
-		return null;
-	}
+	const containerNodeRef = useRef();
+	const nodeRef = useRef();
+
+	const {
+		showMovers,
+		gestures: showMoversGestures,
+	} = useShowMoversGestures( { ref: nodeRef } );
+
+	useExperimentalToolbarPositioning( { ref: containerNodeRef } );
```

Comment:
Any reason for using a "ref" subkey as argument of both hooks, do you expect more config options in the future? Why not pass it as a regular argument?

Reply:
I felt like the explicit key made code easier to follow. If preferred, I can make it a single argument (`ref`)